firmware_configurations/klipper/RealDeuce/MKS-Makerbase/Monster8_v1.0_003/Voron2_Monster8_Config.cfg

```diff
@@ -0,0 +1,531 @@
+##      Uncomment for 350mm build
+#gantry_corners:
+#       -60,-10
+#       410,420
+##      Probe points
+#points:
+#       50,25
+#       50,275
+#       300,275
+#       300,25
+
+#--------------------------------------------------------------------
+speed: 100
+horizontal_move_z: 10
+retries: 5
+retry_tolerance: 0.0075
+max_adjust: 10
+
+#####################################################################
+#   Displays
+#####################################################################
+
+#--------------------------------------------------------------------
+
+[display]
+#! In EXP1/EXP2 (mini12864 LCD Display)
+lcd_type: uc1701
+cs_pin: PE11
+a0_pin: PD10
+rst_pin: PD9
+encoder_pins: ^PE8,^PE9
+click_pin: ^!PE10
+contrast: 63
+#spi_bus: spi1
+spi_software_mosi_pin: PA7
+spi_software_miso_pin: PA6
+spi_software_sclk_pin: PA5
+
+[neopixel mks_mini12864_v3]
+# To control Neopixel RGB in mini12864 display
+pin: PD8
+chain_count: 3
+initial_RED: 0.1
+initial_GREEN: 0.5
+initial_BLUE: 0.0
+color_order: RGB
+
+# Set RGB values on boot up for each Neopixel.
+# Index 1 = display, Index 2 and 3 = Knob
+[delayed_gcode setdisplayneopixel]
+initial_duration: 1
+gcode:
+        SET_LED LED=mks_mini12864_v3 RED=1 GREEN=1 BLUE=1 INDEX=1 TRANSMIT=0
+        SET_LED LED=mks_mini12864_v3 RED=1 GREEN=0 BLUE=0 INDEX=2 TRANSMIT=0
+        SET_LED LED=mks_mini12864_v3 RED=1 GREEN=0 BLUE=0 INDEX=3
+
+[output_pin beeper]
+pin: PB2
+
+#--------------------------------------------------------------------
+
+
+#####################################################################
+#   Macros
+#####################################################################
+
+[gcode_macro G32]
+gcode:
+    BED_MESH_CLEAR
+    G28
+    QUAD_GANTRY_LEVEL
+    G28
+    G0 X175 Y175 Z30 F3600
+
+[gcode_macro M300]
+gcode:
+    # Use a 200ms duration if P is omitted.
+    {% set P = params.P|default(200)|int %}
+    SET_PIN PIN=BEEPER_pin VALUE=1
+    G4 P{P}
+    SET_PIN PIN=BEEPER_pin VALUE=0
+
+[gcode_macro PRINT_START]
+#   Use PRINT_START for the slicer starting script - please customise for your slicer of choice
+gcode:
+    G32                            ; home all axes
+    G1 Z20 F3000                   ; move nozzle away from bed
+
+[gcode_macro PRINT_END]
+#   Use PRINT_END for the slicer ending script - please customise for your slicer of choice
+gcode:
+    M400                           ; wait for buffer to clear
+    G92 E0                         ; zero the extruder
+    G1 E-10.0 F3600                ; retract filament
+    G91                            ; relative positioning
+    G0 Z1.00 X20.0 Y20.0 F20000    ; move nozzle to remove stringing
+    TURN_OFF_HEATERS
+    M107                           ; turn off fan
+    G1 Z2 F3000                    ; move nozzle up 2mm
+    G90                            ; absolute positioning
+    G0  X125 Y250 F3600            ; park nozzle at rear
+    BED_MESH_CLEAR
+
+## Common Temperature Sensors
```

Comment:
remove this section, since the sensors will have the link to the type list in the comments.

Reply:
I'm not sure what you mean by that, but sure.